core/src/main/java/hudson/model/ChoiceParameterDefinition.java

```diff
@@ -57,6 +57,7 @@ private ChoiceParameterDefinition(@NonNull String name, @NonNull List<String> ch
         this.defaultValue = defaultValue;
     }
 
+    // TODO consider switching @DataBoundConstructor to a ChoiceParameterDefinition(String) overload
```

Comment:
Looked complicated to attempt right now: this constructor does not bind the default value for some reason!